test/FilterTest.cpp

```diff
@@ -173,25 +173,30 @@ TEST(Filter, verifySetPrefilterExpressionVariablePairForIndexScanChild) {
   // assigned to the IndexScan child (prefiltering is possible) with Filter
   // construction.
   checkSetPrefilterExpressionVariablePair(
-      qec, Permutation::POS, {Variable{"?x"}, "<p>", Variable{"?z"}},
+      qec, Permutation::POS,
+      {Variable{"?x"}, TripleComponent::Iri::fromIriref("<p>"), Variable{"?z"}},
```

Comment:
I have a shorter helper lambda somewhere in the test helpers that does exactly the lengty .........fromIriref.